docs/toc.js

```diff
@@ -169,9 +169,21 @@ module.exports = {
           type: 'link',
         },
         {
-          pathSegment: 'importing-stories-in-tests',
+          pathSegment: '',
           title: 'Import stories in tests',
-          type: 'link',
+          type: 'menu',
+          children: [
+            {
+              pathSegment: 'end-to-end-testing',
+              title: 'End-to-end tests',
+              type: 'link',
+            },
+            {
+              pathSegment: 'unit-testing',
+              title: 'Unit tests',
+              type: 'link',
+            },
+          ],
```

Comment:
@jonniebigodes — Feel free to disregard this late-in-the-game suggestion, but I think `/writing-tests/stories-in-end-to-end-tests` and `/writing-tests/stories-in-unit-tests` would be more descriptive URLs. The `title` for each could remain the same, though. Remember that the `Import stories in tests` portion doesn't appear in the URL, only in the navigation menu.